src/main/java/com/joyent/manta/monitor/Application.java

```diff
@@ -159,8 +160,19 @@ private static void copyContextToException(final Injector injector,
         });
 
         String message = String.format("%s %d. Additional context is as follows:%s",
-                s, configuration.getJettyServerPort(), messageBuffer);
+                s, jettyServerPort, messageBuffer);
 
         LOG.error(message, e);
     }
+
+    private static int validateJettyServerPort(final String jettyServerPortEnvVariable) {
+        if (jettyServerPortEnvVariable == null) {
+            throw new RuntimeException("Manta Monitor requires env variable JETTY_SERVER_PORT");
```

Comment:
Create a new runtime exception type and don't use the generic `RuntimeException`.